### PR TITLE
Some more Python 3 fixes

### PR DESCRIPTION
**Pull request by takluyver**
This makes some progress in getting the tests running. The next hurdle is the use of `buffer`, which is gone in Python 3. The replacement is `memoryview`, but the API is a bit different. I may look into that tomorrow.

### CODE REVIEW

File: test/test_events_le.py [hunk 1]
```diff
@@ -66,10 +73,10 @@ def setUp(self):
             'type': 173,
             'data': [130, 181, 177, 244, 167, 144, 216, 185, 228, 220, 254, 130, 239, 213, 142, 240, 233, 248, 161, 238, 160, 205, 212, 205, 166, 156, 241, 169, 198, 147, 144],
             }
-        self.evt_bin_0 = '\xad\x82\xb5\xb1' '\xf4\xa7\x90\xd8' \
-            '\xb9\xe4\xdc\xfe' '\x82\xef\xd5\x8e' \
-            '\xf0\xe9\xf8\xa1' '\xee\xa0\xcd\xd4' \
-            '\xcd\xa6\x9c\xf1' '\xa9\xc6\x93\x90'
+        self.evt_bin_0 = b'\xad\x82\xb5\xb1' b'\xf4\xa7\x90\xd8' \
```
Comment (takluyver) [inline test/test_events_le.py]: The diff from here onwards is a find and replace to add the `b` prefix for bytes literals.

**Comment by vasily-v-ryabov** [comment]
Almost everything looks fine to me. Thank you for the efforts!

`QuantifiedCode` issues are all about doc strings for py2/p3 compatibility functions (`_ordb` and `_method`). Although we have many more serious warnings, but I'd prefer to prevent growing the total numbers. The full list is [here](https://www.quantifiedcode.com/app/project/gh:python-xlib:python-xlib).

**Comment by takluyver** [comment]
Voila, docstrings. Also, some more bulk adjustments of bytes literals in the tests.

**Comment by vasily-v-ryabov** [comment]
Great! Merging. Thanks a lot!

P.S. Py3 tests: 414 failures -> 390 failures. This is first visible progress in Py3 pass rate! :+1:

**Comment by takluyver** [comment]
:tada:

**Pull request merged**